.yamllint

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Do we want to have a CI check for linting yaml?

Reply:
It does via the `make lint`, which is part of the `make all` target.